test/http_client_test.exs

```diff
@@ -212,7 +215,7 @@ defmodule Closex.HTTPClientTest do
 
   describe "create_lead/1" do
     test "when close.io is up and we are passed a valid lead payload, creates a lead on CloseIO" do
-      use_cassette "create_lead", match_requests_on: [:request_body] do
```

Comment:
Why did we have these here before?

Reply:
It gave us a bit of extra security in the tests.

Problem is, OTP21 creates maps in a different order. So this test fails on the latest OTP.